Dockerfile-balto

```
@@ -0,0 +1,15 @@
+FROM python:3.8
+
+ARG GH_TOKEN={GH_TOKEN}
+
+COPY requirements.txt /app/requirements.txt
+COPY requirements_elasticc.txt /app/requirements_balto.txt
```

```suggestion
COPY requirements_balto.txt /app/requirements_balto.txt
```
Creo que aquí debería ser `balto` en vez de `elasticc`, no? Lo mismo para los otros Dockerfiles.